src/Neo.VM/Exceptions/IVMException.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// IVMException.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+namespace Neo.VM.Exceptions
+{
+    public interface IVMException
```

Comment:
Making a requirement of for this is irrelevant, just use `Exception`.

Doing this will make the message required for example:
```csharp
public class BadScriptException(string message) : Exception(message) { }
```